### DISCUSSION TITLE
changing install directory in the installer

### DISCUSSION THREAD

**Discussion by FlashlightET** [kaixxx/noScribe#57] [Ideas]
by default, and unchangably, it installs by default to c:\program files (x86)\noScribe. not everyone wants to install things to c:\ or has the space for that (which can be a factor)

**Comment by kaixxx** [comment]
It installs to the default program directory, whereever that is located. 
But your are right, it would be better to make the install directory choosable by the user. I'm in the process of preparing version 0.5 which will be released in a few days. I'll try to include this option.
